submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,197 @@
+				type="checkbox"
+				id="navbar-button"
+			>
+			<div class="focus-div">
+				<img class="image-poup" src="Image/popup-button.png" alt="" >
+			</div>
+
+			<div class="various-app">
+				<div class="app-first">
+					<a class="button-app" href="">
+						<div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/calendar.png" alt="" >
+							</div>
+							<p>Calendar</p>
+						</div>
+					</a>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/mail.png" alt="" ></div>
+							<p>Mail</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/news.png" alt="" ></div>
+							<p>News</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/maps.png" alt="" ></div>
+							<p>Map</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/files.png" alt="" ></div>
+							<p>Files</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/contacts.png" alt="" >
+							</div>
+							<p>Contact</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/tanslation.png" alt="" >
+							</div>
+							<p>Translate</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/videos.png" alt="" >
+							</div>
+							<p>Video</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/search.png" alt="" >
+							</div>
+							<p>Search</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/photos.png" alt="" >
+							</div>
+							<p>Photo</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc"><img src="Image/apps.png" alt="" ></div>
+							<p>Apps</p>
+						</div></a
+					>
+					<a class="button-app" href=""
+						><div class="items-app">
+							<div class="image-bloc">
+								<img src="Image/documents.png" alt="" >
+							</div>
+							<p>Document</p>
+						</div></a
+					>
+				</div>
+
+				<input 
+					class="second-input"
+					type="checkbox"
+					id="second1-navbar-button"
+				>
+				<label for="second1-navbar-button" class="second-navbar-button">
+					<div class="buttom-poup-app"><p>More</p></div>
+				</label>
+
+				<div class="line"></div>
+
+				<div class="app-second">
+					<a class="button-app" href=""
```

Comment:
Avoid leaving an empty `href` attribute, not official convention called fill empty link by symbol `#`.